quadrat/inc/wpcom.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * WordPress.com-specific functions and definitions.
+ *
+ * This file is centrally included from `wp-content/mu-plugins/wpcom-theme-compat.php`.
+ *
+ * @package Quadrat
+ */
+
+
+/**
+ * Enqueue our WP.com styles for front-end.
+ * Loads after theme styles so we can add overrides.
+ */
+function quadrat_wpcom_scripts() {
+	wp_enqueue_style( 'quadrat-wpcom-style', get_stylesheet_directory_uri() . '/inc/wpcom-style.css', array( 'quadrat-styles' ) );
+}
+add_action( 'wp_enqueue_scripts', 'quadrat_wpcom_scripts' );
```

Comment:
Although this isn't used for Blockbase, I wonder if it would be better to add this to Blockbase so that we don't need to add it for all child themes...

Reply:
I did this.